### PR TITLE
Deleted is array

### LINKED ISSUE #8775: Remove deprecated isArray.js
#8526 deprecated `isArray`; it should be removed before the 1.69 release.

### PR DESCRIPTION
**Pull request by YoussefV**
Fixes #8775

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @YoussefV!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: CHANGES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 - The property `Scene.sunColor` has been removed. Use `scene.light.color` and `scene.light.intensity` instead. [#8774](https://github.com/CesiumGS/cesium/pull/8774)
```
Comment (lilleyse) [inline CHANGES.md:8]: Remove the whitespace between the breaking changes entries

File: CHANGES.md [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
 
+- Removed `Cesium.isArray` (from file: `isArray.js`) using native `Array.isArray` instead. [#8779](https://github.com/CesiumGS/cesium/pull/8779)
```
Comment (lilleyse) [inline CHANGES.md:9]: No need to mention `isArray.js`, just mentioning the API change is fine.

```suggestion
- Removed `isArray`. Use the native `Array.isArray` function instead. [#8779](https://github.com/CesiumGS/cesium/pull/8779)
```

**Comment by lilleyse** [comment]
Thanks @YoussefV!

**Pull request merged**